lab-2.py

```diff
@@ -0,0 +1,28 @@
+
+operations_count = 0
+
+def main():
+    ask_again = True
+    while(ask_again):
+        a = input("Enter the numerator: ")
+        b = input("Enter the denominator: ")
+        result = perform_division(a,b)
+        print(result)
+        ask_again = input("Do you want to perform another operation? Enter yes or no: ")
+        if(ask_again == 'yes'):
+            ask_again = True
+        else:
+            ask_again = False
+            print("You performed " + str(operations_count) + " operations, bye!")
+
+
+def perform_division(a,b):
+    global operations_count

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Global variables are dangerous because they can be simultaneously accessed from multiple sections of a program. This frequently results in bugs. Most bugs involving global variables arise from one function reading and acting on the value of a global variable before another function has the chance to set it to an appropriate value. We recommend using a configuration module to mutate global state.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)